### PR TITLE
chore: Updating NPM CPU and Memory requests and limits

### PR DESCRIPTION
**Pull request by jaer-tsun**
**Reason for Change**:

set NPM CPU and memory limits 

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by jaer-tsun** [comment]
waiting on Jorge/Aanand to confirm on CPU/Mem values

**Comment by jaer-tsun** [comment]
ok, they've confirmed, we should be good to go!

File: parts/k8s/addons/1.16/kubernetesmasteraddons-azure-npm-daemonset.yaml [hunk 1]
```diff
@@ -79,8 +79,10 @@ spec:
             privileged: true
           resources:
             requests:
+              cpu: {{ContainerCPULimits "azure-npm-daemonset"}}
```
Comment (jackfrancis) [inline parts/k8s/addons/1.16/kubernetesmasteraddons-azure-npm-daemonset.yaml]: These values are swapped. ContainerCPULimits needs to be under "limits:", and ContainerCPUReqs needs to be under "requests:".

**Comment by jackfrancis** [comment]
`make generate` and then commit generated code changes to pacify linter

**Comment by jackfrancis** [comment]
@jaer-tsun generated code is still not up-to-date

**Comment by jaer-tsun** [comment]
@jackfrancis I've merged with master and ran make generate again but there wasn't any more generated code to push

**Comment by jaer-tsun** [comment]
@jackfrancis I've merged and ran make generate today so it should be up to date

**Comment by jaer-tsun** [comment]
@jackfrancis I updated/merged with master today

**Pull request merged**